electrode-ota-server-routes-acquisition/src/handlers.js

```diff
@@ -16,8 +16,9 @@ const handles = {
         const action = request.url.pathname;
         logger.info(reqFields(request), `${action} request`);
         const snakeCaseRequested = /update_check/.test(action);
+        const snakeCaseParams = /deployment_key/.test(request.url.path);
```

Comment:
in case if `/auth/updateCheck` has snake_case params as only the pathname can be override the default one ?

Reply:
don't know what you mean.  what is deployment_key?

Reply:
It's the `deploymentKey` param when `/updateCheck`

It's the `deployment_key` param when `/update_check`

Reply:
If it's a param, why are you testing against `request.url.path`?
Can you add a test for this.